spec/workers/commit_monitor_handlers/commit_range/graphics_labeler_spec.rb

```diff
@@ -0,0 +1,30 @@
+describe CommitMonitorHandlers::CommitRange::GraphicsLabeler do
+  let(:branch)         { create(:pr_branch) }
+  let(:git_service)    { double("GitService", :diff => double("RuggedDiff", :new_files => new_files)) }
+
+  before do
+    stub_sidekiq_logger
+    stub_settings(:graphics_labeler => {:enabled_repos => [branch.repo.name]})
+    expect_any_instance_of(Branch).to receive(:git_service).and_return(git_service)
```

Comment:
You need to stub https://github.com/ManageIQ/miq_bot/blob/master/app/workers/concerns/branch_worker_mixin.rb#L6-L20

To return the branch you're created here.